.changeset/config.json

```diff
@@ -93,7 +93,8 @@
       "@commercetools-uikit/toggle-input",
       "@commercetools-uikit/tooltip",
       "@commercetools-uikit/utils",
-      "@commercetools-uikit/search-select-input"
+      "@commercetools-uikit/search-select-input",
+      "@commercetools-uikit/search-select-field"
```

Comment:
Also added to the linked changeset list for version bumping. Not sure if it should be done now or after its first release.